src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/ComService.kt

```diff
@@ -94,11 +95,17 @@ class ComService(
       )
     }
 
-    val entityProbationSearchResult = probationSearchApiClient.searchLicenceCaseloadByTeam(
-      body.query,
-      communityApiClient.getTeamsCodesForUser(body.staffIdentifier),
-      probationSearchApiSortBy,
-    )
+    var entityProbationSearchResult: List<ProbationSearchResponseResult>
+
+    if (body.query.isEmpty()) {
+      entityProbationSearchResult = emptyList()
+    } else {
```

Comment:
meh.. would be tempted to push this down into the client... NBD though

Reply:
Will think about this for the final PR for the search ticket